.changeset/cold-ads-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/design-tokens': minor
+---
+
+Adds `headlineBold15`, `headlineMedium15` and `headlineMediumItalic15` to the typography presets
```

Comment:
You want a release to the `@guardian/source` package here. The code from the design tokens package is used to generate the Typography tokens. It is quite confusing... You need to run `make @guardian/source:build-type-presets` then it should generate the new presets in Source

Reply:
Let me know if you want me to go over in more detail

Reply:
Yea this is definitely our fault! It is not clear that updating the tokens does not update Source. I will update the documentation to make this clearer :)

Reply:
We changed how typography presets are made but didn't update the docs. The reason for not directly pulling it from design-tokens in was because really large objects cannot be tree shaken so if u just use 2 typography presets then you still got the whole object.

Reply:
Yea that looks good!